### DISCUSSION TITLE
PU identity provider for telematik access (user-access-ti) not working yet?

### DISCUSSION THREAD

**Discussion by ManuelB** [Digitaler-Impfnachweis/certification-apis#87] [General]
Hi,
I just tried to access the idp with an OpenId-Connect implementation that was working on thursday in the RU infrastructure (https://www.youtube.com/watch?v=C0KfzxP-HzU).

We were using a Secunet PTV3 connector and a pharmacy SMC-B (I know that this should be forbidden but we wanted to give it a try).

We got the following error:
![Screenshot from 2021-06-12 15-59-44](https://user-images.githubusercontent.com/457641/121778695-51a27180-cb98-11eb-9583-ab33535ef285.png)

The realm that we used was the following:
https://id.impfnachweis.info/auth/realms/bmg-ti-certify/protocol/openid-connect/auth

So from my perspective even if we would be using a doctors SMC-B it would still be failing.

Thanks a lot for fixing this as fast as possible.

/Manuel

**Comment by oliver-steinbrecher** [comment]
The content shown in the video has no relation to the RKI solution. In addition the uri used is not valid for the RU environment.

**Reply by mge-medisoftware** [reply]
We get the same error. The uri https://id.impfnachweis.info/auth/realms/bmg-ti-certify/protocol/openid-connect/auth is the production uri documented in https://github.com/Digitaler-Impfnachweis/certification-apis/blob/master/SMCB-Authentication.md

**Reply by ManuelB** [reply]
@oliver-steinbrecher the video shows that it works in the RU. The screenshot shows that it does not work in the PU.

**Comment by ManuelB** [comment]
We can confirm that it, as expected, is not working with a pharmacy.

**Reply by ManuelB** [reply]
Correction. The problem was that the PIN of the SMC was not verified.
